testsuite/MDAnalysisTests/coordinates/test_rdkit.py

```diff
@@ -331,34 +331,44 @@ def test_nan_coords(self):
             mol.GetConformer()
 
     def test_cache(self):
```

Comment:
This test is getting rather convoluted, whilst generally we don't need docstring for tests, I wonder if a quick summary here might help future readers?

i.e. what is being tested, a quick line or two about how you're going about it.